docs/releasing.md

```diff
@@ -8,6 +8,6 @@ To become a maintainer, create your own account on npm.js then ask the owners of
 
 To do this:
 - After merging your changes, run `npm version -f major|minor|patch`. This will create a tagged commit changing the version in `package.json`, and the changes in `CHANGELOG.md`.
-- Run `npm publish`.
+- Run `npm publish --registry=https://registry.npmjs.org/`.
```

Comment:
Be explicit, as not doing so can cause problems publishing to the wrong location.